Assets/Expansion1/Replacements/citystates.xml

```diff
@@ -1,319 +1,319 @@
-              <Stack      ID="EnvoysBonusStack"    Anchor="L,T"  Offset="0,1"    StackPadding="4" />
-              <ScrollBar                          Anchor="R,C"  Offset="0,0"    AnchorSide="O,I"            Style="ScrollVerticalBarAlt" />
-            </ScrollPanel>
-          </Container>
-          <Container			ID="InfluenceArea"																		Size="parent,parent" Hidden="1">
-            <Grid                                                Offset="10,10"  Size="parent-20,40"          Style="CityStateHeader" >
-              <Label                              Anchor="C,C"                                              Style="CityStateHeaderText"          String="{LOC_CITY_STATES_ENVOYS_INFLUENCE_BY_CIVILIZATION:upper}" />
-            </Grid>
-            <ScrollPanel	ID="InfluenceScroll"									Offset="7,60"		Size="parent-15,parent-60"				Vertical="1">
-              <Stack      ID="InfluenceStack"      Anchor="L,T"  Offset="0,1"    StackPadding="4" />
-              <ScrollBar                          Anchor="R,C"  Offset="4,0"    AnchorSide="O,I"            Style="ScrollVerticalBarAlt" />
-            </ScrollPanel>
-          </Container>
-          <Container			ID="QuestsArea"																				Size="parent,parent"	Hidden="1" >
-            <Grid                                                Offset="10,10"  Size="parent-20,40"          Style="CityStateHeader" >
-              <Label                              Anchor="C,C"                                              Style="CityStateHeaderText"          String="{LOC_CITY_STATES_AVAILABLE_QUESTS:upper}" />
-            </Grid>
-            <ScrollPanel	ID="QuestsScroll"											Offset="7,60"		Size="parent-15,parent-60"				Vertical="1" >
-              <Stack      ID="QuestsStack"        Anchor="L,T"  Offset="0,1"    StackPadding="20" />
-              <ScrollBar                          Anchor="R,C"  Offset="-12,0"  AnchorSide="O,I"            Style="ScrollVerticalBarAlt" />
-            </ScrollPanel>
-          </Container>
-          <Container			ID="RelationshipsArea"																				Size="parent,parent"	Hidden="1" >
-            <Grid																								Offset="10,10"	Size="parent-20,40"					Style="CityStateHeader" >
-              <Label															Anchor="C,C"																							Style="CityStateHeaderText"					String="{LOC_CITY_STATES_RELATIONSHIPS:upper}" />
-            </Grid>
-            <ScrollPanel	ID="RelationshipsScroll"											Offset="7,60"		Size="parent-15,parent-60"				Vertical="1" >
-              <Stack Anchor="C,T" StackGrowth="Down" StackPadding="4">
-                <Container Size="parent,auto">
-                  <Grid Style="DividerGrid" Size="parent-8,8" Anchor="C,C" Color="34,48,59,255"/>
-                  <Grid Style="DropShadow4" Size="auto,auto" Anchor="C,C">
-                    <Label String="{LOC_CITY_STATES_CIVILIZATION_HEADER:upper}" Anchor="C,C"  Style="DiplomacyGossipHeader"/>
-                  </Grid>
```

Comment:
The code in this section did not change, the diffing engine is just having a tough time with the spacing it appears